src/less/flex.less

```diff
@@ -292,6 +292,9 @@
 .content--start { .alignContent(flex-start) }
 .content--stretch { .alignContent(stretch) }
 
+.display--block { display: block; }
```

Comment:
these don't seem to fit within `flex.less`. maybe move to a new `display.less` file? 
TBH, i'd be hesitant to encourage these, since it's useful for people to think about the appropriate block element to use for a given scenario and override explicitly when needed.